packages/experiments/tsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
+	"extends": "../../tsconfig.base.json",
+	"compilerOptions": {
+		"rootDir": "src",
+		"declarationDir": "build-types",
+		"checkJs": false
+	},
+	"include": [ "src/**/*" ]
+}
```

Comment:
I had to configure `@wordpress/experiments` to generate type definitions so that I could import it into `@wordpress/components` The definitions generated from the JavaScript seemed fine to me after I added a doc comment to `__dangerousOptInToUnstableAPIsOnlyForCoreModules`. cc. @adamziel